### DISCUSSION TITLE
Does drop_duplicates possible to use?

### DISCUSSION THREAD

**Discussion by tommyhj217** [vaexio/vaex#1995] [Q&A] [answered]
Hi

I checked below.
https://github.com/vaexio/vaex/pull/1623

Now I tired to use drop_duplicate but show below error.
![image](https://user-images.githubusercontent.com/86287388/160961982-71faeb0f-023e-4fb5-9ae5-f37cee4effc1.png)

1. Which version support drop_duplicates()?

Thanks
Hyunjun

**Comment by Hasham04** [comment, marked as answer]
Do this `Vdf.groupby( list of columns to to use, agg={'__hidden_count': vaex.agg.count()}).drop('__hidden_count')`. Hope this helps.

**Reply by tommyhj217** [reply]
Thanks for reply.

**Comment by maartenbreddels** [comment]
There is no drop duplicates yet :) Indeed, the groupby is the current workaround.
